spec/unit/yardstick/ordered_set/left_shift_operator_spec.rb

```diff
@@ -13,4 +13,12 @@
   it { should include(item) }
 
   its(:length) { should be(1) }
+
+  context 'when operation is repeated' do
+    subject do
+      -> { set << item << item }
```

Comment:
I like the expanded `expect` a bit better, but I would probably use something like:

```ruby
it { expect { set << item << item }.to change(set, :entries).from([]).to([item]) }
```